src/Swagger/Serializer/DocumentationNormalizer.php

```diff
@@ -27,6 +27,7 @@
 use ApiPlatform\Core\Metadata\Resource\ResourceMetadata;
 use ApiPlatform\Core\Operation\Factory\SubresourceOperationFactoryInterface;
 use ApiPlatform\Core\PathResolver\OperationPathResolverInterface;
+use ArrayObject;
```

Comment:
I'm not sure that we're doing this. Please wait for someone to confirm that though ^^.

Reply:
My IDE just updated `\ArrayObject` to `ArrayObject` and added the corresponding use statement.

Reply:
We don't do this, please use \ArrayObject please